ecpp/current.ecpp

```diff
@@ -26,43 +26,40 @@
 #include <map>
 #include <string>
 #include <cmath>
-
-#include <fty_proto.h>
-
 #include <sys/types.h>
+
 #include <fty_common_rest_helpers.h>
 #include <fty_common_db_dbpath.h>
-#include <fty_common_mlm_pool.h>
 #include <fty_common_db_asset.h>
 #include <fty_common_asset_types.h>
 #include <fty_common_macros.h>
+#include <fty_proto.h>
 #include <fty_shm.h>
 
 #include "shared/upsstatus.h"
 #include "shared/data.h"
 
-static std::string
-s_os2string(
-        double d) {
+static std::string s_os2string(double d)
+{
     // This is a hack for gdn project - if the key is operatingStatus, the value is
     // converted from number to string using this map
     static std::map<int, std::string> OS2STRING {
         {0, "Unknown"},
-            {1, "None"},
-            {2, "Servicing"},
-            {3, "Starting"},
-            {4, "Stopping"},
-            {5, "Stopped"},
-            {6, "Aborted"},
-            {7, "Dormant"},
-            {8, "Completed"},
-            {9, "Migrating"},
-            {10, "Immigrating"},
-            {11, "Emigrating"},
-            {12, "Snapshotting"},
-            {13, "In_Test"},
-            {14, "Transitioning"},
-            {15, "In_Service"}
+        {1, "None"},
```

Comment:
this can probably be inherited from fty-service-status (https://github.com/42ity/fty-service-status/blob/master/include/fty_service_status.h#L37) though I don't quickly see status strings there

Reply:
Indeed, those constants are defined here:
https://github.com/42ity/fty-service-status-systemd/blob/master/src/systemd.cpp#L29

Example of usage here https://github.com/42ity/fty-service-status-systemd/blob/master/example/src/example.cpp#L49